style.css

```diff
@@ -0,0 +1,102 @@
+.nav__link {
+	cursor: pointer;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Azez1/pullRequest?prid=5948917)